composite/defi-pulse/README.md

```diff
@@ -15,14 +15,25 @@ The ChainlinkAdapter.getAllocations(ISetToken _setToken) should be reimplemented
 The adapter takes the following environment variables:
 
 - `RPC_URL`: Blockchain RPC endpoint to get the needed on-chain data
+- `DATA_PROVIDER`: Data provider to use. Options (Notes):
```

Comment:
Good call. :+1:  It seems very clear to us, because we know these adapters inside out (well, are supposed to at least), but node ops don't have a lot of the context around this - and the clearer the better for them.